### PR TITLE
cp-143: replace app logo with updated one

### PR DESCRIPTION
**Pull request by juanc-escobar**
Updated the logo for the Auth page and the sidebar
Updated the fav icons
updated the styles for the logos. 

Views:

image

image

image

image

image

### CODE REVIEW

File: frontend/src/libs/components/sidebar/styles.module.scss [hunk 1]
```diff
@@ -1,6 +1,10 @@
 .image {
-  width: 56px;
-  height: 56px;
+  width: 59px;
+  height: 59px;
+}
+
+.image img {
```
Comment (fshabanov) [inline frontend/src/libs/components/sidebar/styles.module.scss]: Do not use global tags in styles. Rule B10 in quality criteria

**Review by fshabanov — CHANGES_REQUESTED** [review]

**Review by dmytro-kucherenko — DISMISSED** [review]

**Review by OlenaYatseniuk — DISMISSED** [review]
Everything looks good. 
Just noticed that you accidentally linked an issue to your pr. So, please, remove it.

**Review by iamAlinaaa — DISMISSED** [review]

**Comment by juanc-escobar** [comment]
@OlenaYatseniuk  did you mean this one:

@[juanc-escobar](https://github.com/juanc-escobar) juanc-escobar linked an issue [17 hours ago](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/162#event-10264928098) that may be closed by this pull request
https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/issues/143

I got assigned an issue so I created a task for the issue, if you meant another thing can you please tell me a bit more specific so I can find it?

Thank You.

File: frontend/src/libs/components/sidebar/styles.module.scss (hunk 1 above)
Comment (juanc-escobar) [reply frontend/src/libs/components/sidebar/styles.module.scss]: @fshabanov I updated the code, thanks for your review.

**Comment by liza-veis** [comment]
@juanc-escobar, @OlenaYatseniuk is right, we should unlink issues from PRs as it will automatically close the issue after merging

File: frontend/src/assets/img/logos/logos.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export { default as logoM } from './logo-m.svg';
```
Comment (liza-veis) [inline frontend/src/assets/img/logos/logos.ts]: remove this file, no ts files in assets, also remove logos folder and just keep them in assets/img

File: frontend/src/libs/components/sidebar/sidebar.tsx [hunk 3]
```diff
@@ -18,8 +18,12 @@ const Sidebar: React.FC<Properties> = ({ routes }) => {
     <div className={styles['sidebar']}>
       <nav className={styles['nav']}>
         <Link to={AppRoute.ROOT}>
-          <span className={styles['image']}>
-            <img src={logo} alt="logo" />
+          <span className={styles['logo']}>
+            <img
+              src={logoS}
+              className={styles['logo-img']}
+              alt="CalmPal Logo blue"
```
Comment (liza-veis) [inline frontend/src/libs/components/sidebar/sidebar.tsx]:
```suggestion
              alt="CalmPal logo"
```

File: frontend/src/libs/components/sidebar/styles.module.scss [hunk 4]
```diff
@@ -1,6 +1,10 @@
...
-  width: 56px;
-  height: 56px;
+.logo {
+  width: 59px;
+  height: 59px;
+}
+
+.logo-img {
+  width: 59px;
 }
```
Comment (liza-veis) [inline frontend/src/libs/components/sidebar/styles.module.scss]: let's add these attributes to image itself
```suggestion
```

**Review by liza-veis — CHANGES_REQUESTED** [review]

File: frontend/src/pages/auth/auth.tsx [hunk 5]
```diff
@@ -42,11 +43,20 @@ const Auth: React.FC = () => {
...
+        <picture className={styles['logo']}>
+          <source
+            media="(width < 1023px)"
+            srcSet={logoM}
+            type="image/svg+xml"
+          />
+          <img
+            src={logoXl}
+            className={styles['logo-img']}
+            alt="white logo of CalmPal size XL"
```
Comment (liza-veis) [inline frontend/src/pages/auth/auth.tsx]:
```suggestion
            alt="CalmPal logo"
```

**Review by liza-veis — CHANGES_REQUESTED** [review]

File: frontend/src/pages/auth/styles.module.scss [hunk 6]
```diff
@@ -71,26 +73,23 @@
...
+  .auth-name {
+    width: 280px;
+    text-align: start;
+  }
+
+  .auth-name .secondary {
+    display: none;
+  }
+
+  .auth-name .logo .logo-img {
```
Comment (dmytro-kucherenko) [inline frontend/src/pages/auth/styles.module.scss]: According to the rule B11 it's allowed maximum 2 level selector's nesting, not 3

File: frontend/src/libs/components/sidebar/styles.module.scss (hunk 4 above)
Comment (juanc-escobar) [reply frontend/src/libs/components/sidebar/styles.module.scss]: I'm confused can I use the global tag  for this one or did you mean to delete the span and give a classname to the image?

B10. No global tag styles. Only class selectors or a cascade are used to set styles.
Global styles are only available in global style files (scaffolding.css, app.css, reset.css (🥲), etc).

I will update the PR like that while you can check the question and if it needs to change I will change it again.

File: frontend/src/libs/components/sidebar/sidebar.tsx [hunk 7]
```diff
@@ -18,13 +18,7 @@ const Sidebar: React.FC<Properties> = ({ routes }) => {
...
       <nav className={styles['nav']}>
         <Link to={AppRoute.ROOT}>
-          <span className={styles['logo']}>
-            <img
-              src={logoS}
-              className={styles['logo-img']}
-              alt="CalmPal Logo blue"
-            />
-          </span>
+          <img src={logoS} className={styles['logo']} alt="CalmPal logo" />
```
Comment (vladyslavdobychin) [inline frontend/src/libs/components/sidebar/sidebar.tsx]: You might want to use the Icon component from cp-55 and cp-61, it's a universal component for rendering svgs

File: frontend/src/libs/components/sidebar/styles.module.scss (hunk 4 above)
Comment (vladyslavdobychin) [reply frontend/src/libs/components/sidebar/styles.module.scss]: I think Liza meant either of two: 
1. If it is SVG file, you can define its size right in the svg: 
Screenshot 2023-09-04 at 12 45 40

2. You can pass inline styling properties to the image: 
`` 
Something like this

Comment (liza-veis) [reply frontend/src/libs/components/sidebar/styles.module.scss]: yes, @vladyslavdobychin are right

File: frontend/src/libs/components/sidebar/sidebar.tsx (hunk 7 above)
Comment (liza-veis) [reply frontend/src/libs/components/sidebar/sidebar.tsx]: it's not for all SVGs, only for icons, so it's ok here

File: frontend/src/pages/auth/styles.module.scss [hunk 8]
```diff
@@ -71,26 +73,23 @@
...
-    background: var(--blue-300);
-    border-radius: 14px;
+  .auth-name {
+    width: 280px;
+    text-align: start;
+  }
+
+  .auth-name .secondary {
+    display: none;
+  }
```
Comment (fshabanov) [inline frontend/src/pages/auth/styles.module.scss]: Remove duplicate style selectors. `.auth-name` and `.auth-name .secondary` are used multiple times

**Review by fshabanov — CHANGES_REQUESTED** [review]

File: frontend/src/pages/auth/styles.module.scss [hunk 9]
```diff
@@ -33,19 +29,20 @@
 .auth-name {
   position: relative;
   z-index: 2;
-  margin-bottom: 150px;
+  width: 420px;
```
Comment (liza-veis) [inline frontend/src/pages/auth/styles.module.scss]: avoid fixed width
```suggestion
```

File: frontend/src/pages/auth/styles.module.scss [hunk 10]
```diff
@@ -56,11 +53,13 @@
     gap: 20px;
   }
 
-  .auth-name {
-    margin: 0;
+  .logo-img {
+    width: 290px;
```
Comment (liza-veis) [inline frontend/src/pages/auth/styles.module.scss]: same
```suggestion
```

**Review by liza-veis — CHANGES_REQUESTED** [review]

**Review by OlenaYatseniuk — DISMISSED** [review]

**Review by OlesiaTr — DISMISSED** [review]

File: frontend/src/pages/auth/styles.module.scss (hunk 9 above)
Comment (juanc-escobar) [reply frontend/src/pages/auth/styles.module.scss]: Hi, @Liza-Veis 

I want to clarify some doubts, Do you mean I should not use fixed width for images and containers? what about the rest of the items like gaps, margins, and paddings, Should I use fixed units or should I change them?

and which properties am I allowed to use for the rest of the items?
px
em
rem
ch
%
vw
vh
clamp()

I usually work with relative units in almost all the CSS, but I received this task with styles already implemented and it was using fixed units so I used the same type of styles and built upon that. 

so can you please clarify to me what are the CSS unit's limitations so I can have a better understanding of future tasks?

Comment (juanc-escobar) [reply frontend/src/pages/auth/styles.module.scss]: I removed the widths from CSS for the code I added. and also leaved the gaps and margins in px. here is a demo:
logo

**Review by OlesiaTr — DISMISSED** [review]

**Review by v0ldemar01 — DISMISSED** [review]

**Review by liza-veis — APPROVED** [review]

**Pull request merged**